src/net/java/sip/communicator/impl/protocol/sip/ConfigHeaders.java

```diff
@@ -160,6 +164,15 @@ static void attachConfigHeaders(
 
                 Header h = request.getHeader(name);
 
+                if (name.equals(SIPHeaderNames.ROUTE) 
+                    || name.equals("P-Asserted-Identity"))
+                {
+                    if (seqNumber != 1)
+                    {
```

Comment:
These changes seems unrelated to the PR, could you remove them and squash to commits.